lib/init/npm-utils.js

```diff
@@ -121,12 +121,12 @@ function check(packages, opt) {
 
     ["dependencies", "devDependencies"].forEach(key => {
         if (opt[key] && typeof fileJson[key] === "object") {
-            deps = deps.concat(Object.keys(fileJson[key]));
+            deps.add(Object.keys(fileJson[key]));
```

Comment:
I think this method needs to be called with each dependency individually, rather than the array- `Set` objects can store arrays as members.

Maybe try something like this?

```js
Object.keys(fileJson[key]).forEach(dep => deps.add(dep));
```

Reply:
oops :sweat_smile: